libpod/container_internal.go

```diff
@@ -683,6 +684,21 @@ func (c *Container) makeBindMounts() error {
 		c.state.BindMounts["/run/.containerenv"] = containerenvPath
 	}
 
+	// Add secrets mounts
+	mountsFiles := []string{secrets.OverrideMountsFile, secrets.DefaultMountsFile}
```

Comment:
Don't like the []string here - do we really think we're going to be adding another one of these? If we're worried about code deduplication, make a helper function